de/faq.html

```diff
@@ -48,10 +76,33 @@ <h2>Ich möchte mit meinem Coworking Space auch dabei sein, wie melde ich ihn an
       </p>
     </article>
     <article>
-      <h2>Mein Coworking Space macht eine Veranstaltung, kannst du sie teilen?</h2>
+      <h2>Mein Coworking Space macht eine Veranstaltung zum Festival, wie kann ich sie teilen?</h2>
       <p>
         Wir teilen deine Veranstaltung gerne in unserem Veranstaltungskalender und auf unseren Social-Media-Kanälen.
-        Vorraussetzung ist du hast deinen Space für das Festival angemeldet.
+        Vorraussetzung ist du hast deinen Space für das Festival angemeldet. Wir schreiben dir rechtzeitig nach
```

Comment:
`Voraussetzung ist, dass du mindestens einen Space für das Coworking Festival angemeldet hast.`

(ein `r` zu viel bei Voraussetzung; Satzbau)

Vielleicht wäre ein Link zum Anmeldeformular hier noch cool?

Reply:
gibt es noch nicht. Bzw. für die Spaces ist es direkt in dem Absatz dadrüber.